### DISCUSSION TITLE
Custom brand icons

### DISCUSSION THREAD

**Discussion by mselmasry** [matt8707/ha-fusion#530] [Q&A] [answered]
How can I use custom brand icons (https://elax46.github.io/custom-brand-icons/) inside fusion ?

**Comment by matt8707** [comment, marked as answer]
https://icon-sets.iconify.design/cbi/

**Reply by mselmasry** [reply]
Thank you @matt8707 this helps.
I also have some fontawesome pro icons which i access now by adding them to a custom icons folder in www anyway to access those ?

**Reply by matt8707** [reply]
Yes, instead of `mdi:lightbulb` use `/local/custom_icons/icon.svg`
